source/util/ilist_node.h

```diff
@@ -81,6 +99,51 @@ template <class NodeType>
 inline IntrusiveNodeBase<NodeType>::IntrusiveNodeBase()
     : next_node_(nullptr), previous_node_(nullptr), is_sentinel_(false) {}
 
+template<class NodeType>
+inline IntrusiveNodeBase<NodeType>::IntrusiveNodeBase(
+    const IntrusiveNodeBase&) {
+  next_node_ = nullptr;
+  previous_node_ = nullptr;
+  is_sentinel_ = false;
+}
+
+template<class NodeType>
+inline IntrusiveNodeBase<NodeType>& IntrusiveNodeBase<NodeType>::operator=(
+    const IntrusiveNodeBase&) {
+  assert(!is_sentinel_);
+  if (IsInAList()) RemoveFromList();
```

Comment:
The preference is to always use braces for control flow bodies, even if they're only one line.  I don't think the codebase is very consistent about that, but @dneto0 has requested changes on that before.

Reply:
done